recipes/server.rb

```diff
@@ -17,15 +17,24 @@
 # limitations under the License.
 #
 
+Chef::Recipe.send(:include, MariaDB::Helper)
 case node['mariadb']['install']['type']
 when 'package'
-  include_recipe "#{cookbook_name}::repository"
+  if use_os_native_package?(
```

Comment:
I think that here, there a possible error happening. If someone want set the flag to use native os package, but is on some debian flavor... Then it will try to use redhat like code. A case node['platform'] would be great. Perhaps someone wil add the code for native ubuntu mariadbpackage use.